### PR TITLE
Updated formatting issues in Kubeflow's Azure installation.

### PR DESCRIPTION
**Pull request by mohona17**
---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/816)

### CODE REVIEW

File: content/docs/azure/deploy/install-kubeflow.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ kfctl generate all -V
 kfctl apply k8s -V
 ```
 
-    * ${KFAPP} - the name of a directory where you want Kubeflow configurations to be stored. This directory is created when you runkfctl init. If you want a custom deployment name, specify that name here. The value of this variable becomes the name of your deployment. The value of this variable cannot be greater than 25 characters. It must contain just the directory name, not the full path to the directory. The content of this directory is described in the next section.
+* ${KFAPP} - the name of a directory where you want Kubeflow configurations to be stored. This directory is created when you runkfctl init. If you want a custom deployment name, specify that name here. The value of this variable becomes the name of your deployment. The value of this variable cannot be greater than 25 characters. It must contain just the directory name, not the full path to the directory. The content of this directory is described in the next section.
```
Comment (sarahmaddox) [inline content/docs/azure/deploy/install-kubeflow.md]: The numbering is incorrect in the preview:
https://deploy-preview-816--competent-brattain-de2d6d.netlify.com/docs/azure/deploy/install-kubeflow/

**Comment by sarahmaddox** [comment]
@mohona17 Please would you rebase this PR so that you're the only author of the commit? It's a very small set of changes, so it doesn't seem necessary to have multiple authors. Having a single author will make the automated CLA check work, and will remove the necessity for repository managers to check all the email addresses in the commit.

**Comment by mohona17** [comment]
Tried rebasing but ran into merge conflicts. Decided to start a new PR.

**Pull request closed without merging**